third-exam-third-task/src/main/java/writer/JsonWriterStrategy.java

```diff
@@ -0,0 +1,22 @@
+package writer;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import pojo.Employee;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+
+public class JsonWriterStrategy implements Strategy {
+    @Override
+    public void write(List<Employee> employeeList, String fileName) {
+        ObjectMapper objectMapper = new ObjectMapper();
+        try {
+            objectMapper.writeValue(new File(fileName), employeeList);
+            System.out.println("Data written successfully to employees.json");
```

Comment:
please check java.util.logging and start using logging libraries - this one is easy to set up (no setup needed, so it's really easy :D)